src/components/Input/CheckBox/CheckBox.styles.js

```diff
@@ -47,8 +51,10 @@ export const CheckDivStyling = styled.div`
   opacity: 0;
   color: ${getThemeValue("primary", "base")};
   transition: opacity 0.3s ${constants.easing.elastic};
-
-  .checkbox__checked & {
+  ${({ isTopAligned }) =>
+    isTopAligned
+      ? "align-items: flex-start;"
+      : "align-items: center;"} .checkbox__checked & {
```

Comment:
missing new line before `.checkbox__checked &`

Reply:
need to remove `align-items: center` at line 46
https://github.com/ticketmaster/aurora/pull/662/files#diff-1e69fb4d33cb039e8e2e2cd952cbe6ccR46

Reply:
Fixed, thanks